course/page/code.py

```diff
@@ -43,6 +43,9 @@
         get_editor_interaction_mode)
 from course.constants import flow_permission
 
+# DEBUGGING SWITCH: False for 'spawn container', True for 'static container'
+USE_STATIC_CONTAINER = False
```

Comment:
Sorry, I copied it from your originally description at [here](https://github.com/inducer/relate/blob/master/course/page/code.py#L98) and I actually don't know if there are some kind of `static container`. I'll modify that.

Reply:
Oh. 😊 

In that case, a "static container" was one that i spawned manually by hand so that I could watch the output of runpy in the terminal. It's still bad terminology, but it's less bad when it's my fault, right? *cough* :)